scripts/sipify.py

```diff
@@ -678,6 +678,10 @@ def read_line():
             f"OVR: {CONTEXT.is_override_or_make_private} CLSS: {CONTEXT.actual_class}/{len(CONTEXT.classname)} :: {new_line}"
         )
 
+    # SIP doesn't like Qt 6.4 u""_s or ""_L1 literals
+    new_line = re.sub(r'u("(?:(?:\\.|[^"\\])*)")_s', r"QStringLiteral( \1 )", new_line)
+    new_line = re.sub(r'("(?:(?:\\.|[^"\\])*)")_L1', r"QLatin1String( \1 )", new_line)
```

Comment:
This will also change `QLatin1Char`s to `QLatin1String`s, right? Could that be a problem with SIP?

Reply:
oh sorry I read your comment too quickly.   No the regex only catches "foo"_L1.   I haven't found an instance where our .h files contains a 'x'_L1

Reply:
Ah, right, I missed the `"` vs `'`.

However, there are 3 cases of `QLatin1Char` in .h files, I don't know if it matters (`qgis.h`, `qgslayertree.h` and `postgresconn.h` which is not SIPed)

Reply:
> However, there are 3 cases of `QLatin1Char` in .h files

ok, I see that prepare_commit_code_fixup.py was lacking the substitution for QLatin1Char('x') to 'x'_L1. I've added it, and the corresponding undoing in sipify.py